recipes/rsatoolbox/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "rsatoolbox" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rsatoolbox-{{ version }}.tar.gz
+  sha256: 2d091cbaa33373bf9da4df5ca8d127f0e427431a3db726076090ab2d54fe1213
+
+build:
+  skip: true  # [py<37 or py>310 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython # >=3.0.0a11
+    - numpy >=1.21.2
+    - scipy
+    - setuptools
+    - setuptools-scm
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - {{ pin_compatible('scipy') }}
+    - pandas
+    - scikit-learn
+    - scikit-image
+    - matplotlib-base
+    - h5py
+    - joblib
+    - tqdm
+    - coverage
```

Comment:
Coverage is a very test-specific dependence, I'd be surprised that this is necessary for correct operation of the package?

Reply:
Good catch! removed.